### PR TITLE
fix: Removed un-necessary workflows

### LINKED ISSUE #28447: Delete the unnecessary github actions workflows
There's a number of github actions workflows that used to be used in the past but aren't being used anymore or have been replaced with something else. All such workflows should be deleted. Your task is to create a PR that removes all such workflows which you think aren't being used anymore. One good way to find out which ones aren't being used is going through the [workflows](https://github.com/unifyai/ivy/actions) and seeing which ones were last triggered months ago and so on.
The decision on whether to remove a particular workflow is slightly subjective, so you should do a first pass through all the workflows and creating a PR that removes them, we'll then discuss further on the PR if there's any others that should be removed or ones which shouldn't be removed.
Feel free to reach out for any questions, thanks!

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# PR Description
Removed un-necessary workflows.

## Related Issue
Closes #28447

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [x] Did pre-commit not fail on any check?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by Sai-Suraj-27** [comment]
Hey, @vedpatwardhan. I removed all the workflows that I think are not useful now. I'm not 100% sure about the following 5, but I removed them, so please let me know if they are useful, I will add them again.
1. [label_discussion_comment.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/label_discussion_comment.yml)
2. [manual-tests.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/manual-tests.yml)
3. [manual-tests-pr.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/manual-tests-pr.yml)
4. [run-failing-tests.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/run-failing-tests.yml)
5. [test-ivy-cron-gpu.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/test-ivy-cron-gpu.yml)

Thank you.

File: .github/workflows/github_release.yml [hunk 1]
```diff
@@ -1,34 +0,0 @@
-name: github-release
-on:
-  workflow_dispatch:
```
Comment (vedpatwardhan) [inline .github/workflows/github_release.yml:3]: this workflow should be retained

File: .github/workflows/pypi.yml [hunk 2]
```diff
@@ -1,32 +0,0 @@
-name: pypi
-on:
```
Comment (vedpatwardhan) [inline .github/workflows/pypi.yml:2]: this workflow should be retained

**Review by vedpatwardhan — COMMENTED** [review]
> Hey, @vedpatwardhan. I removed all the workflows that I think are not useful now. I'm not 100% sure about the following 5, but I removed them, so please let me know your opinion on these.
> 
> 1. [label_discussion_comment.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/label_discussion_comment.yml)
> 2. [manual-tests.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/manual-tests.yml)
> 3. [manual-tests-pr.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/manual-tests-pr.yml)
> 4. [run-failing-tests.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/run-failing-tests.yml)
> 5. [test-ivy-cron-gpu.yml](https://github.com/unifyai/ivy/blob/main/.github/workflows/test-ivy-cron-gpu.yml)
> 
> Thank you.

Along with these 5, there's a few more that I've flagged that should be retained, rest can be removed. Thanks @Sai-Suraj-27 😄

File: .github/workflows/github_release.yml (hunk 1 above)
Comment (Sai-Suraj-27) [reply .github/workflows/github_release.yml:3]: Done 👍🏻

File: .github/workflows/pypi.yml (hunk 2 above)
Comment (Sai-Suraj-27) [reply .github/workflows/pypi.yml:2]: Done 👍🏻

**Review by vedpatwardhan — APPROVED** [review]
lgtm! Feel free to merge the PR, thanks @Sai-Suraj-27 😄

**Pull request merged**